nginx.conf

```diff
@@ -1,10 +1,21 @@
-worker_processes      1;
+worker_processes      auto;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /tmp/nginx.pid;
 
 events {
   worker_connections  1024;
 }
 
 http {
+  proxy_temp_path /tmp/proxy_temp;
+  client_body_temp_path /tmp/client_temp;
+  fastcgi_temp_path /tmp/fastcgi_temp;
+  uwsgi_temp_path /tmp/uwsgi_temp;
+  scgi_temp_path /tmp/scgi_temp;
+
+  include /etc/nginx/conf.d/*.conf;
```

Comment:
Why we're including the default config here?

Reply:
see my answerr above